### ISSUE TITLE
Request: BIP39-style mnemonic output type

### ISSUE THREAD

**Issue by shawnz** [dmshaw/paperkey#2]
Hi there, thank you for the very useful tool.

I think it would be helpful if paperkey additionally supported a BIP39-style output format where the data would be encoded as words from a word list rather than hex characters. This might make transcription of the data easier.

**Comment by jonathancross** [comment]
Note: BIP39 is for a 256bit seed (or smaller), so 4096bit RSA keys would need a LOT of words.

ECC or EdDSA keys would work great though.  If a larger wordlist (like that of the [EFF Diceware project](https://www.eff.org/deeplinks/2016/07/new-wordlists-random-passphrases)) were used (12.9 bits per word), we would get a slightly shorter mnemonic than that one in BIP39.

**Comment by dmshaw** [comment]
Yes, that was my concern about using a wordlist here: hex can be more difficult to type in, but at least fits on a reasonable number of sheets of paper :)

Note that you can always use a wordlist, even with the current version of paperkey, by using "`--output-type raw`".  This is intended for piping to a program to handle different representations of data (QR code, wordlists, etc).

**Comment by jonathancross** [comment]
Yes, ~ [320 words](https://loremipsum.io/generator/?n=320&t=w) might be a bit much. But it would still be a great option to have for the Elliptic Curve keys.  A backup which relies on 2 different pieces of software is less reliable, and given that Paperkey is apparently pre-installed in TailsOS, it would be ideal if it just worked out of the box.

A few nice features for BIP39 which might help here as well:

- uses extra bits as a checksum -- so errors can be detected.
- Because the dictionary is fixed, users only need to type in the beginning of each word and it is auto completed.
- spelling is nearly impossible to mess up.

Note: Using a mnemonic means we don't need to trust our printer, [evil USB cables](https://mg.lol/blog/badusb-cables/), etc.
